podcasts/Notifications/NotificationsCoordinator.swift

```diff
@@ -77,7 +85,14 @@ class NotificationsCoordinator {
 
     static let shared: NotificationsCoordinator = NotificationsCoordinator()
 
-    var timeIntervalStep: TimeInterval = 24.hours
+    var onboardingTimeIntervalStep: TimeInterval = 24.hours
+    var reEngagementTimeIntervalStep: TimeInterval = 1.week
+    var ignoreScheduleHours: Bool = false
+
+    enum Constants {
```

Comment:
⛏️  Can the Constants be private?

```suggestion
    private enum Constants {
```

Reply:
Done here: https://github.com/Automattic/pocket-casts-ios/pull/3006/commits/8080a0a78ae83f607cbe6320eeaa1bfd63656cd6